bears/php/PHPMessDetectorBear.py

```diff
@@ -19,7 +19,9 @@ class PHPMessDetectorBear:
     """
 
     LANGUAGES = {'PHP'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='phpmd')}
+    REQUIREMENTS = {AnyOneOfRequirements([
+                    DistributionRequirement(apt_get='phpmd'),
```

Comment:
(just do whatever you have time for; the rest can be added after this is merged.)